test/drivers/test_revd2.cc

```diff
@@ -80,6 +112,25 @@ class TestREVD2 : public ::testing::Test
         norm_A = lapack::lange(Norm::Fro, m, m, all_data.A_cpy.data(), m);
     }
 
+    template <typename T, typename RNG>
+    static void uplo_computational_helper(REVD2UploTestData<T>& all_data) {
+        auto m = all_data.dim;
+        // Filling the upper-triangular matrix
+        blas::syrk(Layout::ColMajor, Uplo::Upper, Op::Trans, m, m, 1.0, all_data.work.data(), m, 0.0, all_data.A_u.data(), m);
+        // Filling the lower-triangular matrix
+        blas::syrk(Layout::ColMajor, Uplo::Lower, Op::Trans, m, m, 1.0, all_data.work.data(), m, 0.0, all_data.A_l.data(), m);
```

Comment:
@TeachRaccooon, don't use a second call to syrk. Just copy to the other triangle.

Reply:
That is, copy the upper triangle of ``A_u`` into the lower triangle of ``A_l``, instead of writing to the lower triangle of ``A_l`` with ``syrk``.

Reply:
Fixed.